hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapMergePolicyTest.java

```diff
@@ -77,30 +69,38 @@ public void killAllHazelcastInstances() {
     @Parameterized.Parameter
     public ReplicatedMapMergePolicyTestCase testCase;
 
+    private TestHazelcastInstanceFactory factory;
```

Comment:
there is no instance termination in this test class since it is being handled by the `HazelcastTestSupport`, as you say. This one only creates a factory to create instances in the test method. What is the issue here?